runtime/src/test/resources/tailcall/runtime/graphql/test-modify-interface-field.graphql

```diff
@@ -0,0 +1,38 @@
+#> server-sdl
+schema {
+  query: Query
+}
+
+interface IA {
+  a: String
+}
+
+type B implements IA {
+  a: String @modify(name: "x")
+  b: String
+}
+
+type Query {
+  bar: B
+}
+
+#> client-sdl
+schema {
+  query: Query
+}
+
+interface IA {
+  a: String
+}
+
+type B implements IA {
+  b: String
+  x: String
+}
+
+type Query {
+  bar: B
+}
+
+#> client-error
+# - [B, a, @modify] Cannot use modify on field a because it is implemented from interface IA
```

Comment:
- [ ] Can we move the error to the top of the file?
- [ ] Can we make sure that invalid graphql specs are in a separate folder and are not mixed with the valid ones, where are are asserting server-SDL with produced client-SDL.